R/utils.R

```diff
@@ -227,10 +228,16 @@ save_chunkdata <- function(chunkdata, write_inputs = FALSE, outputs_dir =
   # Create directory if necessary, and remove any previous outputs
   dir.create(outputs_dir, showWarnings = FALSE, recursive = TRUE)
   unlink(file.path(outputs_dir, "*.csv"))
+  dir.create(xml_dir, showWarnings = FALSE, recursive = TRUE)
+  unlink(file.path(xml_dir, "*.xml"))
 
   for(cn in names(chunkdata)) {
     cd <- chunkdata[[cn]]
-    if(!isTRUE(identical(NA, cd))) {   # NA means an optional file that wasn't found
+    if(FLAG_XML %in% get_flags(cd)) {
+      # TODO: worry about absolute paths?
```

Comment:
The code here, following the same strategy for outputting data to CSV, is adjusting the XML file path set by the user by some base path.  One reason such a feature is required is because the working directory for the tests is different when just running the package.  So when test cases are running the base path is set to `../../`.

This wouldn't work if the file paths set by the user are absolute.  That being said file paths are typically "modeltime.xml" and this isn't likely going to be a problem at all.  Maybe the best thing to do is just drop the comment and not worry about it.